Automattic/sensei
# Do not show quiz notice if it is the quiz page

Fixes #6268 

### Changes proposed in this Pull Request

* Don't show quiz notice when in quiz page in learning mode.

### Testing instructions

- Go to a quiz and complete it
- Visit the quiz page again
- There should be no notice which shows the quiz grade and a 'View quiz' link.
